# Metabolite Damage and Metabolite Damage Control in Plants

Annual Review of Plant Biology
DOI: 10.1146/annurev-arplant-043015-111648
Andrew D. Hanson,1 Christopher S. Henry,3,4 Oliver Fiehn,5 and Valérie de Crécy-Lagard2

---

### ABSTRACT

It is increasingly clear that (*a*) many metabolites undergo spontaneous or enzyme-catalyzed side reactions in vivo, (*b*) the damaged metabolites formed by these reactions can be harmful, and (*c*) organisms have biochemical systems that limit the buildup of damaged metabolites. These damage-control systems either return a damaged molecule to its pristine state (metabolite repair) or convert harmful molecules to harmless ones (damage preemption). Because all organisms share a core set of metabolites that suffer the same chemical and enzymatic damage reactions, certain damage-control systems are widely conserved across the kingdoms of life. Relatively few damage reactions and damage-control systems are well known. Uncovering new damage reactions and identifying the corresponding damaged metabolites, damage-control genes, and enzymes demands a coordinated mix of chemistry, metabolomics, cheminformatics, biochemistry, and comparative genomics. This review illustrates the above points using examples from plants, which are at least as prone to metabolite damage as other organisms.

---

### INTRODUCTION

The terms “metabolite damage” and “metabolite damage control” may look unfamiliar, but they rest on several simple and uncontroversial ideas. First, chemically reactive metabolites—which is most of them—will undergo spontaneous chemical side reactions in vivo ([32](javascript:void(0);)). Second, enzymes that catalyze side reactions in vitro—again, most of them—will do the same in vivo ([13](javascript:void(0);), [77](javascript:void(0);)). Third, damage products formed by metabolic side reactions will be useless, if not toxic, and hence unwanted ([7](javascript:void(0);)). And last, cells need systems to stop such damage products from accumulating ([7](javascript:void(0);), [28](javascript:void(0);)). These ideas now seem self-evident, and classical papers (e.g., [50](javascript:void(0);), [69](javascript:void(0);), [81](javascript:void(0);)) show that the twentieth-century pioneers of metabolic biochemistry thought they were. However, metabolite damage and its control have been largely overlooked for the last 50 years.

This is all the more surprising because metabolite damage and its control closely parallel DNA and protein damage and repair ([**Figure 1a**](javascript:popRef('f1'))), which are well researched ([101](javascript:void(0);), [103](javascript:void(0);)). Why, then, has metabolite damage ([**Figure 1b**](javascript:popRef('f1'))) been so little studied? One reason may be the past prevalence of a paradigm that saw metabolic pathways as purpose driven and essentially error free ([32](javascript:void(0);)). There being no place in this unfortunate paradigm for disorderly metabolite chemistry, imperfect enzymes, and their metabolic repercussions, it was easy to dismiss these topics as trivia. There were, and still are, also practical disincentives to performing metabolite damage research ([64](javascript:void(0);)). First, damaged metabolites are often low abundance, labile, and commercially unavailable, making them easy to miss in analytical procedures and hard to identify if found. Nonavailability also discourages their use as biochemical substrates. Second, knocking out damage-control genes often has only subtle effects—i.e., they impact fitness, not survival ([64](javascript:void(0);)). Subtle phenotypes let mutants slip through classical genetic screens and generally hinder genetic studies.

![img](http://www.annualreviews.org/na101/home/literatum/publisher/ar/journals/content/arplant/2016/arplant.2016.67.issue-1/annurev-arplant-043015-111648/20160412/images/medium/pp670131.f1.gif)

> **Figure 1**  Damage-control systems for metabolites are analogous to those for macromolecules. The red arrows indicate damage reactions, and the blue arrows indicate various types of damage-control reactions; at the bottom, the solid black arrow indicates a normal enzyme reaction, and the dashed black arrow indicates a relatively slow spontaneous reaction. (*a*) It is well known that DNA and proteins are damaged by spontaneous reactions and enzymatic mistakes, and that enzymatic damage-control systems counter this damage. (*b*) The same applies to metabolites. Metabolite damage-control systems either repair the damage or preempt it. Directed overflow is a subcategory of damage preemption. Modified from Reference [64](javascript:void(0);).

Whatever its causes, the neglect of metabolite damage is over. Interest is burgeoning in the chemical side reactions of metabolites ([15](javascript:void(0);), [52](javascript:void(0);),[64](javascript:void(0);)) and even more in the side reactions mediated by metabolic enzymes ([10](javascript:void(0);), [54](javascript:void(0);), [75](javascript:void(0);)) and in systems that limit metabolite damage ([64](javascript:void(0);), [101](javascript:void(0);), [103](javascript:void(0);)). Fascinatingly, some damage-control systems have proven to be conserved across kingdoms of life, presumably because organisms share a core set of metabolites, and these suffer the same damage reactions always and everywhere ([64](javascript:void(0);)). Work involving plants has been at the forefront of much of this progress.

Accordingly, here we first explain what chemical and enzymatic metabolite damage reactions are and how metabolite damage-control systems work, giving specific cases from plants. Next, we trace a connection between metabolite damage and the huge number of unidentified peaks in metabolomics profiles, and describe progress in systematizing information on damage reactions and in predicting damage products. Lastly, we suggest that hundreds of plant genes of unknown function may encode metabolite damage-control functions, and discuss how cross-kingdomcomparative genomics can help predict these functions. Plants are good subjects for comparative genomics of metabolite damage because plastids as well as mitochondria are damage hot spots and have inherited damage-control systems from their prokaryotic progenitors.

---

### CHEMICAL DAMAGE

Russian biochemist Alexey Golubev captured the essence of chemical damage to biomolecules in the six words “nothing of chemistry disappears in biology” ([31](javascript:void(0);), p. 6). Thus, metabolites that are chemically reactive and unstable in vitro remain so in vivo, and can actually become more unstable in vivo because they are exposed to other reactive molecules ([3](javascript:void(0);), [32](javascript:void(0);)). Many metabolic intermediates and end products thus constantly face spontaneous chemical damage ([32](javascript:void(0);), [52](javascript:void(0);), [64](javascript:void(0);)). The damage processes are of three types: unimolecular reactions (racemization, rearrangement, elimination, and photolysis), reactions with water or oxidants, and reactions with other metabolites (addition and condensation) ([**Figure 2a**](javascript:popRef('f2'))). Of these, only the reactions of certain metabolites with oxidants have been much studied in plants ([21](javascript:void(0);), [26](javascript:void(0);)).

![img](http://www.annualreviews.org/na101/home/literatum/publisher/ar/journals/content/arplant/2016/arplant.2016.67.issue-1/annurev-arplant-043015-111648/20160412/images/medium/pp670131.f2.gif)

> **Figure 2**  Chemical damage to metabolites. (*a*) Common spontaneous chemical reactions that metabolites may undergo, depending on their structure. (*b*) Illustrative first-order decay curves and half-lives (*t*½) under physiological conditions for reactive intermediates in key metabolic pathways: the tryptophan synthesis intermediate *N*-(5′-phosphoribosyl)-anthranilate (PRA); the arginine precursor carbamoyl phosphate (CP); the glycolytic intermediate 1,3-diphosphoglycerate (DPG); and the purine, pyrimidine, and histidine precursor 5-phosphoribosyl 1-pyrophosphate (PRPP). (*c*) The chemical reactions responsible for the rapid decay of these four intermediates. The PRA and DPG reactions are hydrolyses, and the CP and PRPP reactions are eliminations.

It may come as a shock to realize just how chemically reactive a metabolite can be.[**Figure 2b**](javascript:popRef('f2')) gives four striking examples of highly unstable intermediates from central metabolic pathways (there are many more such cases). Under physiological conditions (neutral pH, ∼37°C), these intermediates have half-lives of only 3.5–58 min ([12](javascript:void(0);), [25](javascript:void(0);), [70](javascript:void(0);), [105](javascript:void(0);)). Such half-lives become shorter still in leaves hitting temperatures of ∼50°C ([61](javascript:void(0);)) because chemical reaction rates roughly double per 10°C rise in temperature.

Although many different classes of metabolites are chemically reactive, some—the usual suspects—are more so than others and are hence particularly damage prone. These metabolites are also potentially damaging to other molecules. The examples shown in [**Figure 2b**](javascript:popRef('f2'))represent two such metabolite classes: acyl phosphates (carbamoyl phosphate and 1,3-diphosphoglycerate) and sugar phosphate derivatives, both of which can undergo elimination or hydrolysis ([**Figure 2c**](javascript:popRef('f2'))). Note that the cyanate produced by breakdown of carbamoyl phosphate is an indiscriminate carbamoylating agent, so the initial chemical damage event leads to others. Such knock-on effects are common in metabolite damage chemistry. Other usual-suspect damage-prone metabolites include reactive carbonyls (e.g., methylglyoxal, triose phosphates, and fumarate) ([93](javascript:void(0);)); imines (e.g., 2-aminoacrylate) ([17](javascript:void(0);)); thiols (e.g., cysteine and coenzyme A) ([109](javascript:void(0);)); and cofactors such as flavins, pyridine nucleotides, thiamin phosphates, pyridoxal and pyridoxamine phosphates, folates, and *S*-adenosylmethionine ([64](javascript:void(0);)). That cofactors are seriously damage prone reflects that they are, by nature, molecules whose normal role is to react with other metabolites.

---

### ENZYMATIC DAMAGE

It is now clear that most if not all enzymes make mistakes (i.e., that they are “promiscuous” or “sloppy”) and that some make more mistakes than others ([10](javascript:void(0);), [13](javascript:void(0);), [54](javascript:void(0);)). Promiscuity—the ability to catalyze slow, secondary reactions differing from the normal ones—is not the same as broad specificity; enzymes that are broadly specific (e.g., alkaline phosphatase) have the ability to catalyze the same reaction on various physiological substrates at similar rates ([10](javascript:void(0);), [102](javascript:void(0);)). The side activities of promiscuous enzymes are of two types. In the first type, an enzyme catalyzes its normal reaction on an abnormal (noncanonical) substrate. This is termed substrate promiscuity (right reaction, wrong substrate). In the second type, which is less common, an enzyme catalyzes an abnormal reaction on its normal substrate. This is termed catalytic promiscuity (right substrate, wrong reaction). Both types of side activity damage metabolites by converting them to aberrant products; recent examples of both types are given below.

The side reactions of promiscuous enzymes proceed at far lower rates (∼102–107-fold slower) than their normal, physiological reactions ([102](javascript:void(0);)). However, this slowness is deceptive because damage-product formation can be significant over time, especially if the enzyme is in a high-flux pathway or the noncanonical substrate is at high concentration. Two examples from mammals underscore these points. The glycolytic enzyme triose phosphate isomerase forms the toxic side product methylglyoxal at only 1/5,000 of the rate of the isomerase reaction, but daily methylglyoxal production in rats can nevertheless reach 0.4 μmol per gram of tissue ([86](javascript:void(0);)). Likewise, mitochondrial malate dehydrogenase catalyzes α-ketoglutarate reduction to l-2-hydroxyglutarate ∼107 times less efficiently than its normal reaction, oxaloacetate reduction. But because malate dehydrogenase is abundant and the mitochondrial α-ketoglutarate level is ∼100 times that of oxaloacetate, the side activity can form several grams of l-2-hydroxyglutarate per day in a human adult ([100](javascript:void(0);)).

Lastly, it should be noted that enzymatic and chemical damage reactions are not mutually exclusive—i.e., some enzymatic side reactions of metabolites can also occur spontaneously. The example of NAD(P)H hydrate formation covered below is one such case.

---

### DAMAGE-CONTROL MECHANISMS

Damage-control systems can be classified in various ways and have been given various names. A classification based on that of Linster et al. ([64](javascript:void(0);)) is shown in [**Figure 1b**](javascript:popRef('f1')) and outlined in the remainder of this section, along with some commonly used terms. Illustrative examples follow in the next section.

#### Repair

Repair deals with damage after it has happened. It simply returns a chemically or enzymatically damaged molecule to its original state via one or more enzymatic reactions ([**Figure 1b**](javascript:popRef('f1'))). When the damage is enzymatic—i.e., an enzyme makes an unwanted side product—and the repair system reconverts this side product to the metabolite from which it came, the term metabolite proofreading can be used, by analogy to the proofreading activities of DNA polymerases and aminoacyl-tRNA synthetases ([100](javascript:void(0);)).

#### Damage Preemption

Preemption stops damage from happening. It involves enzymes that either (*a*) convert a potentially harmful chemical or enzymatic side product to a benign one or (*b*) convert a normal but reactive and damaging intermediate to its normal product much faster than can occur spontaneously ([**Figure 1b**](javascript:popRef('f1'))). The first case, which has been called “house cleaning” ([28](javascript:void(0);)), is the best-known type of metabolite damage control. The second case—in which a preemption enzyme speeds up an insufficiently fast chemical reaction—is turning out to be crucial in the hydrolysis (quenching) of reactive enamine/imine intermediates ([17](javascript:void(0);), [71](javascript:void(0);)). The hydrolysis of these intermediates was formerly thought to be purely spontaneous because their half-life for chemical hydrolysis is 4 min or less ([62](javascript:void(0);)). A lesson here is that reactions that proceed quite fast chemically may still require acceleration by a damage preemption enzyme.

#### Directed Overflow

Directed overflow also stops damage from happening. It is a form of preemption in which an excess of a normal metabolite that can potentially give rise to harmful products is enzymatically diverted to one or more benign products before it does so ([27](javascript:void(0);), [85](javascript:void(0);)) ([**Figure 1b**](javascript:popRef('f1'))). The benign product can be as simple as the dephosphorylated version of a phosphometabolite. Directed overflow heads off damage even earlier in the event chain than other preemptive mechanisms do, and may be concentrated in pathways that lack feedback regulation mechanisms to prevent the buildup of intermediates ([85](javascript:void(0);)).

---

### EXEMPLARS FROM PLANTS

This section describes four iconic cases of metabolite damage and its repair or preemption, all drawn from plants. These examples are not confined to plants, however; the fact that they are also present in other organisms was crucial to their discovery, which in each case involved cross-kingdom comparative genomics. The comparative genomics trails that led to these discoveries are retraced below (see the section Metabolite Damage and Genomics: The Unknown Gene Problem). [**Table 1**](javascript:popRef('t1'))summarizes 14 more cases of metabolite damage and damage control in plants, most of which also occur in other kingdoms of life. It is worth noting that all four cases described below were recognized in the past two years, and that 10 of the cases in [**Table 1**](javascript:popRef('t1')) are similarly recent. This emphasizes the accelerating tempo of work on metabolite damage. 

[![Data table](http://www.annualreviews.org/templates/jsp/_style4/_ar/images/dummy_table_thumb.gif)](javascript:popRef('t1'))

**Table 1**  Additional cases of metabolite damage and damage-control enzymes in plants

#### An Epimerase-Dehydratase Duo Repairs NAD(P)H Hydrates

The damage reaction in this case is both enzymatic and chemical. NADH and NADPH are converted to hydrates (NADHX and NADPHX, respectively) either via a side reaction of glyceraldehyde 3-phosphate dehydrogenase (GAPDH) or spontaneously ([1](javascript:void(0);)). NADHX and NADPHX exist as a mixture of C-6 *R* and *S* epimers ([**Figure 3a**](javascript:popRef('f3'))), and because they inhibit various dehydrogenases, they cannot be left to accumulate in cells ([67](javascript:void(0);)).

![img](http://www.annualreviews.org/na101/home/literatum/publisher/ar/journals/content/arplant/2016/arplant.2016.67.issue-1/annurev-arplant-043015-111648/20160412/images/medium/pp670131.f3.gif)

> **Figure 3**  Examples of metabolite damage and its repair or preemption. Damage reactions are shown in red; repair or preemption reactions are shown in blue. (*a*) NADH and NADPH are converted to hydrates (NADHX and NADPHX, respectively) either via a side reaction of glyceraldehyde 3-phosphate dehydrogenase (GAPDH) or spontaneously. An ATP-dependent NAD(P)HX dehydratase returns the *S* form of the hydrates to NAD(P)H, and an NAD(P)HX epimerase interconverts the *R* and *S* forms. (*b*) Ribulose 1,5-bisphosphate (RuBP) is epimerized at the 3 position by a side reaction of Rubisco, yielding xylulose 1,5-bisphosphate (XuBP), a Rubisco inhibitor. A selective phosphatase (XuBPase) hydrolyzes XuBP to xylulose 5-phosphate (Xu5P), a normal Calvin-Benson cycle intermediate. The dashed gray arrow indicates multiple steps in the Calvin-Benson cycle. (*c*) Intermediates 1 and 2 of riboflavin synthesis are reactive and spontaneously form Maillard products, 5-phosphoribosylamine (PRA), and other harmful compounds. An *N*-glycosidase (COG3236) hydrolyzes excess intermediates 1 and 2 to ribose 5-phosphate (R5P) and a pyrimidine (HTP or DAU), which are fairly harmless. (*d*) Threonine dehydratase converts threonine or serine to highly reactive enamine/imine tautomers. The serine product ([2](javascript:void(0);)-aminoacrylate) can inactivate pyridoxal 5′-phosphate (PLP) enzymes via adduct formation with the PLP cofactor. RidA enzymes preempt such damage by hydrolyzing the imines to 2-oxo acids; the hydrolysis also occurs spontaneously (*dashed black arrow*), but too slowly to prevent enamine/imine damage to enzymes.

The repair of NAD(P)HX is a two-part process. An ATP-dependent NAD(P)HX dehydratase reconverts the *S* epimer of the hydrates to NAD(P)H, and an NAD(P)HX epimerase facilitates interconversion of the *R* and *S* epimers ([9](javascript:void(0);), [67](javascript:void(0);), [73](javascript:void(0);)) ([**Figure 3a**](javascript:popRef('f3'))). Knocking out the dehydratase in *Arabidopsis*leads to massive (up to 40-fold) accumulation of the hydrates ([73](javascript:void(0);)). Both the dehydratase and the epimerase are targeted in plants to multiple subcellular compartments via use of alternative translation start sites ([9](javascript:void(0);), [73](javascript:void(0);)). This multiple targeting of the repair enzymes presumably reflects the cell-wide nature of NAD(P)HX formation and the need to counter it wherever it occurs.

#### A Selective Bisphosphatase Preempts Blockage of Rubisco Activity

The potential damage source here is a minor side reaction (misfire reaction) catalyzed by Rubisco ([**Figure 3b**](javascript:popRef('f3'))). This reaction epimerizes the Rubisco substrate ribulose 1,5-bisphosphate at the 3 position to give xylulose 1,5-bisphosphate, which is a potent Rubisco inhibitor and must consequently be disposed of ([5](javascript:void(0);), [82](javascript:void(0);)). Potential harm to Rubisco is preempted by a selective sugar phosphatase that hydrolyzes xylulose 1,5-bisphosphate to xylulose 5-phosphate, a normal Calvin-Benson cycle intermediate ([5](javascript:void(0);)) ([**Figure 3b**](javascript:popRef('f3'))).

#### An *N*-Glycosidase Safely Directs Excess Intermediates Out of the Riboflavin Pathway

In this case, the potential sources of damage are the first two intermediates of the riboflavin synthesis pathway ([**Figure 3c**](javascript:popRef('f3'))), which may accumulate because riboflavin synthesis is not feedback regulated ([27](javascript:void(0);)). Both intermediates are highly reactive and break down spontaneously and rapidly to create harmful compounds that likely include Maillard products and 5-phosphoribosylamine ([24](javascript:void(0);), [27](javascript:void(0);), [66](javascript:void(0);)).The damage that these products could otherwise inflict is preempted by an*N*-glycosidase (COG3236) that hydrolyzes excess intermediates 1 and 2 to ribose 5-phosphate and pyrimidine moieties, which are relatively harmless ([27](javascript:void(0);)). The *N*-glycosidase thus mediates a directed overflow type of damage preemption—i.e., it safely diverts a dangerous excess of intermediates out of the pathway.

#### Hydrolases Speed Up Conversion of Reactive Imines to 2-Oxo Acids

In this instance, the potential damage agents are reactive enamine/imine tautomers ([**Figure 3d**](javascript:popRef('f3'))). These are the normal initial products of the pyridoxal 5′-phosphate (PLP) enzyme threonine dehydratase acting on threonine or serine, of other PLP enzymes, and of amino acid oxidase ([17](javascript:void(0);),[71](javascript:void(0);)); they were formerly thought to hydrolyze spontaneously to the final 2-oxo acid products ([62](javascript:void(0);)). The enamines/imines, particularly 2-aminoacrylate, can inactivate PLP enzymes via adduct formation with the PLP cofactor ([17](javascript:void(0);), [72](javascript:void(0);)). Although enamines/imines spontaneously hydrolyze very quickly, this reaction (half-time of 4 min or less) is insufficient to avoid damage to the aminotransferase required for branched-chain amino acid synthesis and other PLP enzymes ([17](javascript:void(0);), [72](javascript:void(0);)). RidA enzymes preempt damage from enamines/imines by facilitating their hydrolysis to 2-oxo acids ([**Figure 3d**](javascript:popRef('f3'))). RidA enzymes can also preempt damage from the extremely reactive compound carbamoyl phosphate ([**Figure 2b,c**](javascript:popRef('f2'))), but the biochemical mechanism has yet to be determined ([71](javascript:void(0);)).

---

### METABOLITE DAMAGE, CHEMICAL SPACE, AND METABOLOMICS

The concept of chemical space—the ensemble of all possible molecules—is very important for thinking about metabolite damage and the analysis of damage products. The first point to grasp is that chemical space is vast, and metabolic pathways occupy very little of it. For instance, current chemical databases comprise more than 40 million structures ([53](javascript:void(0);)), but there are only 15,000 metabolic substrates and products in the KEGG (Kyoto Encyclopedia of Genes and Genomes) database ([51](javascript:void(0);)). Metabolic pathways are thus narrow, well-lit tunnels of known chemistry boring through a huge, dark matrix of unknown chemistry ([**Figure 4**](javascript:popRef('f4'))). Damage reactions are excursions from the light of the tunnels out into the adjacent darkness. The damage reactions of central carbon and energy pathways, which are far better characterized than most (but nowhere near completely), illustrate how much damage traffic may cross the tunnel walls of metabolism as a whole ([**Figure 4**](javascript:popRef('f4'))). One way to identify damaged metabolites should therefore be to search the chemical space outside pathways using metabolomics, looking for unknown peaks and then identifying them. This is much easier said than done ([11](javascript:void(0);)).

![img](http://www.annualreviews.org/na101/home/literatum/publisher/ar/journals/content/arplant/2016/arplant.2016.67.issue-1/annurev-arplant-043015-111648/20160412/images/medium/pp670131.f4.gif)

> **Figure 4**  The chemical space occupied by central carbon and energy pathways (in-pathway chemical space) and part of the vast chemical space surrounding these pathways (out-pathway chemical space) that metabolite damage reactions begin to penetrate. Note that the total out-pathway space is at least 1,000 times the size of the in-pathway space. The pathway intermediates and cofactors are shown in black within the blue cylinder of the in-pathway space; chemical or enzymatic damage reactions and the resulting damage products are in red in the out-pathway space. Abbreviations: CoA-SH, coenzyme A; CoA-SO3, CoA sulfonate; CoA-SS, CoA disulfide; CoA-SSO3, CoA *S*-sulfonate; DHAP, dihydroxyacetone phosphate; GAP, glyceraldehyde 3-phosphate; GSH, glutathione; α-NAD(P)H, α anomer of NAD(P)H; NAD(P)HX, NAD(P)H hydrate; UQ, ubiquinone. The chemical and enzymatic damage reactions come from References [20](javascript:void(0);), [45](javascript:void(0);), [46](javascript:void(0);), [86](javascript:void(0);), [96](javascript:void(0);), and [117](javascript:void(0);) and from C. Lerma-Ortiz, J.G. Jeffryes, A.J.L. Cooper, T.D. Niehaus, A.M. Thamm, et al., manuscript in review.

#### Metabolomics Peaks May Well Be Damage Products but Are Hard to Identify

Plant metabolomics profiles from accurate mass liquid chromatography–mass spectrometry (LC-MS) instruments can include thousands of unknown (i.e., unannotated) signals ([30](javascript:void(0);)), and many of these could be damage products ([23](javascript:void(0);), [64](javascript:void(0);)). But it is extremely hard to establish the identities of the unknown molecules that give rise to these signals. Simply matching their observed masses against the predicted masses of the more than 40 million chemical database structures is of little use because, even when the experimental data are sufficient to define a unique elemental composition (which they often are not), many molecules within chemical space have the same elemental composition and hence the same mass ([55](javascript:void(0);)). Nor is mass fragment spectral matching a catch-all solution, because (*a*) it relies on comparisons to spectra of authentic standards and (*b*) the major mass spectral libraries (ReSpect, MassBank, Metlin, and NIST14) collectively contain only 40,000 experimentally determined electrospray tandem mass spectra ([56](javascript:void(0);), [59](javascript:void(0);))—and these are obviously of known compounds, not unknown damage products. Fortunately, cheminformatics can help.

#### Cheminformatics Can Predict Metabolite Damage Products

Cheminformatic approaches can help identify metabolite damage products in two basic ways: (*a*) by predicting the mass spectra and retention times of hypothetical database structures that can be matched with those of unknown metabolomics peaks, and (*b*) by predicting potential damage products of known metabolites using (bio)chemical principles (i.e., by exploring chemical space).For the first approach, heuristic rules for bond cleavages in specific compound classes can be derived from experimental spectra ([56](javascript:void(0);), [57](javascript:void(0);), [87](javascript:void(0);),[107](javascript:void(0);), [111](javascript:void(0);)), and retention times can be modeled for a specific chromatographic method via quantitative structure-property relationships ([98](javascript:void(0);)). However, the great diversity of damaged metabolites may challenge these structural cheminformatic approaches, making it necessary to use more powerful strategies such as accurate calculation of bond energies and the likelihood that bond dissociations will actually occur using first-principle or quantum chemistry approaches. At least for electron ionization mass spectra, this is now feasible ([36](javascript:void(0);)). Such first-principle computational advances now need to be extended to electrospray collision-induced fragmentation, as used in LC-MS-based metabolomics.In the second approach, BNICE (Biochemical Network Integrated Computational Explorer) ([38](javascript:void(0);)) or other cheminformatics tools ([44](javascript:void(0);), [58](javascript:void(0);), [79](javascript:void(0);)) enable high-throughput generation of hypothetical metabolite damage chemistry. The first step is to encode generic reaction rules that capture (bio)chemical damage mechanisms (e.g., decarboxylation or nonenzymatic carbamoylation) using databases of example reactions ([79](javascript:void(0);)) or expert chemical knowledge ([38](javascript:void(0);), [44](javascript:void(0);), [58](javascript:void(0);)). These rules are then applied to databases of known metabolites, e.g., PlantCyc ([115](javascript:void(0);)) or AraCyc ([116](javascript:void(0);)), to propose all possible damage reactions and products, as is done in the MINE (Metabolic In Silico Network Expansion) databases for enzymatic reactions ([49](javascript:void(0);)) and the Chemical Damage MINE for spontaneous reactions (C. Lerma-Ortiz, J.G. Jeffryes, A.J.L. Cooper, T.D. Niehaus, A.M. Thamm, et al., manuscript in review). The rules can also be applied in a targeted way to individual metabolites to assess which damage reactions they are most likely to undergo ([27](javascript:void(0);)).Damage predictions can be sharpened by integrating predicted metabolite and reaction properties into genome-scale metabolic models. For example, group contribution methods to estimate the standard Gibbs free energy change of reactions ([48](javascript:void(0);)) can identify damage reactions that are far from thermodynamic equilibrium in cells and thus likely to go toward completion ([41](javascript:void(0);), [60](javascript:void(0);)), and thermodynamics-based metabolic flux analysis can predict flux through reactions and hence possible concentrations of damage products ([40](javascript:void(0);)). More generally, in conjunction with genome-scale metabolic models ([22](javascript:void(0);), [92](javascript:void(0);)), thermodynamics-based metabolic flux analysis and other flux balance analysis methods can identify indispensable metabolites (C. Lerma-Ortiz, J.G. Jeffryes, A.J.L. Cooper, T.D. Niehaus, A.M. Thamm, et al., manuscript in review). The damage reactions of such pivotal metabolites are particularly high-priority targets for research.

---

### METABOLITE DAMAGE AND GENOMICS: THE UNKNOWN GENE PROBLEM

#### Many Genes of Unknown Function Likely Encode Metabolite Damage-Control Enzymes

Given that a multitude of metabolites can undergo chemical or enzymatic damage, and that the search for damage-control enzymes has only just begun, it is likely that many such enzymes—and the genes that specify them—remain to be discovered. Here, we estimate how many of these genes are in the *Arabidopsis* genome based on the number of genes of unknown function, the percentage of these genes that code for enzymes, and the percentage of these that perform damage control.Although the genomes of plants and other organisms clearly encode many enzymes of unknown function ([37](javascript:void(0);), [43](javascript:void(0);)), establishing the exact percentage of unknown genes in a genome is problematic because the definitions of known and unknown functions vary. In early genome annotations, functions as vague as “ATPase” or “transporter” were counted as known, whereas best current practice as implemented by UniProt (Universal Protein Resource) ([99](javascript:void(0);)) is more stringent and uses various types of evidence (sequence similarity or experimental data) ([83](javascript:void(0);), [84](javascript:void(0);)). However, some of the functions called by UniProt would still be classified as unknown by the most stringent definitions, in which gene function must be experimentally validated in vitro or in vivo. The problem of function calling is compounded by the overannotation of paralogous families in which a gene that has been erroneously given a specific function is taken as known when in fact it is not ([91](javascript:void(0);)). Estimates of unknown gene percentages are thus necessarily rough.With this disclaimer in place, to estimate how many enzymes of unknown function are present in *Arabidopsis* and how many of these could be involved in metabolite damage control, we analyzed the genomes of both*Arabidopsis* and *Escherichia coli*. By various estimates, the *Arabidopsis*genome encodes ∼10,000 enzymes ([74](javascript:void(0);)) ([**Supplemental Table 1**](http://www.annualreviews.org/doi/suppl/10.1146/annurev-arplant-043015-111648)). To estimate how many of these are of unknown function, we combined different UniProt filters ([**Supplemental Table 1**](http://www.annualreviews.org/doi/suppl/10.1146/annurev-arplant-043015-111648)). This gave numbers between 1,300 and 8,600, depending on the strictness of the criteria used to define a function as known. Note that the lower bound (1,300) is certainly an underestimate because it unreasonably assumes that Gene Ontology molecular function assignments always correspond to biological function unless the enzyme is labeled as uncharacterized.In 1998, *E. coli* genes whose functions were at that time unknown were given “Y” names, e.g., *ygfZ* ([89](javascript:void(0);)). In May 2015, we extracted 1,441 Y genes from EcoGene ([118](javascript:void(0);)), of which 1,324 remained after eliminating pseudogenes. By combining UniProt and SEED ([80](javascript:void(0);)) database annotations with manual analysis, we found that 231 of these Y genes have had solid functions assigned to them since 1998. Of these 231, manual analysis indicated that 93 (or 40%) encode enzymes, which is a little above the automated prediction of ∼30% enzymes in the current protein space ([106](javascript:void(0);)). Of the 93, 14 were damage-control enzymes, all from conserved families ([**Supplemental Table 2**](http://www.annualreviews.org/doi/suppl/10.1146/annurev-arplant-043015-111648)). Thus, 15% of the previously unknown enzymes in *E. coli* whose functions have recently been elucidated have roles in metabolite damage control.Projecting this 15% value from *E. coli* onto the estimated 1,300–8,600 unknown-function enzymes in *Arabidopsis* implies that there are between 200 and 1,300 undiscovered metabolite damage-control enzymes in this and other plants. Even taking the conservative lower bound of ∼200 still gives a sense of how much damage-control machinery is still “missing” and, consequently, why a focused effort to identify metabolite damage-control genes is needed. Fortunately, as discussed next, metabolite damage-control enzymes are often conserved between plants and other organisms, so comparative genomics provides a powerful way to predict their functions.

#### Discovering Damage-Control Functions via Comparative Genomics

Comparative genomics, particularly between bacteria and plants, has been a royal road to discovery of metabolite damage-control genes and enzymes and is likely to remain so in future ([64](javascript:void(0);)). Below, we illustrate how this discovery process works by revisiting the four exemplars of metabolite damage and its repair or preemption described above.

##### NAD(P)HX epimerase.

This case demonstrates the value of protein-protein fusions. The partners in such fusions—which have aptly been called “Rosetta stones” ([95](javascript:void(0);))—nearly always have related functions. After the yeast NAD(P)HX dehydratase gene was found via classical biochemistry and cloning ([67](javascript:void(0);)), its homologs in bacteria and mammals ([67](javascript:void(0);)) and then plants ([9](javascript:void(0);), [73](javascript:void(0);)) were readily identified. In *E. coli* and most other bacteria, the dehydratase gene proved to be fused to an N-terminal domain of unknown function ([**Figure 5a**](javascript:popRef('f5'))). Because the dehydratase was known to act on the *S*epimer of NAD(P)HX but not on the *R* epimer, the N-terminal domain was predicted to be an NAD(P)HX epimerase ([67](javascript:void(0);)). Epimerase homologs were then identified in yeast, mammals, and plants ([9](javascript:void(0);), [67](javascript:void(0);), [73](javascript:void(0);)).

![img](http://www.annualreviews.org/na101/home/literatum/publisher/ar/journals/content/arplant/2016/arplant.2016.67.issue-1/annurev-arplant-043015-111648/20160412/images/medium/pp670131.f5.gif)

> **Figure 5**  Examples of comparative genomic evidence predicting damage-control functions. (*a*) The yeast NAD(P)HX dehydratase gene was cloned via classical biochemistry and found to have homologs in bacteria, mammals, and plants. In most bacteria, the dehydratase gene is fused to an N-terminal domain of previously unknown function. This domain was predicted and then shown to be NAD(P)HX epimerase. Epimerase homologs were then identified in eukaryotes, including plants. (*b*) The *cbb*operon of the photosynthetic bacterium *Rhodobacter sphaeroides* and similar operons in other bacteria encode Rubisco subunits (*cbbL* and*cbbS*), Rubisco activase (*cbbX*), phosphoglycolate phosphatase (*cbbZ*), and a haloacid dehalogenase (HAD) family phosphatase (*cbbY*) of previously unknown function. CbbY and its *Arabidopsis* homolog (AtCbbY) were predicted and shown to have a function related to the Calvin-Benson cycle (hydrolysis of the Rubisco misfire product xylulose 1,5-bisphosphate). (*c*) The first enzyme of riboflavin synthesis (RibA) in *Vibrio*species and the third enzyme (RIBR) in plants are fused to a domain of previously unknown function (COG3236). This domain was predicted and shown to have a damage-control role (hydrolysis of excess riboflavin intermediates). (*d*) In taxonomically diverse bacteria, genes encoding RidA family proteins cluster with genes of carbamoyl phosphate metabolism, including those encoding aspartate carbamoyltransferase (*ACT*), ornithine carbamoyltransferase (*argI*), acetylornithine deacetylase (*argE*), and carbamate kinase (*CK*). This clustering predicted a damage-control role for RidA in carbamoyl phosphate metabolism, which was subsequently confirmed by genetic evidence. Additional abbreviation: TP, targeting peptide.

##### Xylulose 1,5-bisphosphate phosphatase.

This case shows how the chromosomal context of a bacterial gene can suggest its function and that of its plant homolog. Prokaryotic genes that consistently cluster together on the chromosome are likely to have related functions ([37](javascript:void(0);)). The *cbb* operon of the photosynthetic bacterium *Rhodobacter sphaeroides* and similar operons in other bacteria were shown to encode Rubisco subunits, Rubisco activase, phosphoglycolate phosphatase, and a haloacid dehalogenase (HAD) family phosphatase (CbbY) of unknown function ([29](javascript:void(0);)) ([**Figure 5b**](javascript:popRef('f5'))). The clustering of CbbY with genes related to the Calvin-Benson cycle and the chloroplastic location of the *Arabidopsis*CbbY homolog suggested a role in the Calvin-Benson cycle, and membership in the HAD family indicated that this role would involve dephosphorylation ([5](javascript:void(0);)). CbbY was subsequently shown to hydrolyze the Rubisco misfire product xylulose 1,5-bisphosphate to xylulose 5-phosphate ([5](javascript:void(0);)).

##### COG3236 *N*-glycosidase.

This case reemphasizes how informative protein fusions can be. The first enzyme of riboflavin synthesis (RibA) in *Vibrio*species and related bacteria was found to be fused (either N or C terminally) to a domain of unknown function, COG3236 ([27](javascript:void(0);)) ([**Figure 5c**](javascript:popRef('f5'))). Remarkably, a COG3236 protein was also found to be fused to a different enzyme of riboflavin synthesis (RIBR) in green plants from algae to angiosperms ([**Figure 5c**](javascript:popRef('f5'))). These fusions from different kingdoms of life pointed strongly to a function related to the riboflavin pathway, and bothcheminformatic evidence ([27](javascript:void(0);)) and distant homology to proteins predicted to depurinate ADP-ribose ([16](javascript:void(0);)) suggested that COG3236 is an *N*-glycosidase. COG3236 proteins were subsequently shown to hydrolyze the *N*-glycosidic bond in the first two intermediates of riboflavin synthesis ([27](javascript:void(0);)).

##### RidA and carbamoyl phosphate.

This is a second case in which gene clustering was crucial to a functional prediction. It was first observed that, besides clustering with PLP-related genes, genes encoding RidA family proteins also clustered in taxonomically diverse bacteria with pyrimidine and arginine metabolism genes, particularly those related to carbamoyl phosphate, notably aspartate and ornithine carbamoyltransferases, acetylornithine deacetylase, and carbamate kinase ([71](javascript:void(0);)) ([**Figure 5d**](javascript:popRef('f5'))). Given the extreme instability and reactivity of carbamoyl phosphate and its breakdown product cyanate ([105](javascript:void(0);)), this clustering pattern predicted a damage-control role for RidA in carbamoyl phosphate metabolism. Genetic support for such a role was subsequently obtained ([71](javascript:void(0);)).

---

### THE ENVIRONMENTAL AND GENOMIC CONTEXT

Two final key points are that plants are at least as prone to metabolite damage as other organisms, and that metabolite damage can potentially reach up into the genome and epigenome.

Plant proneness to metabolite damage stems from their exposure to solar radiation, to sharp and sometimes large temperature changes, and to varying water content, all of which can promote both chemical and enzymatic side reactions. Thus, the temperature, concentration, pH, and ionic-strength dependences of chemical and enzymatic reactions tend to drive up damage reaction rates during heat stress and water deficit. Furthermore, because enzyme specificity can also depend on physicochemical parameters such as temperature and type of metal cofactor ([8](javascript:void(0);), [65](javascript:void(0);), [112](javascript:void(0);)) and metabolite pool sizes change drastically in stressed plants ([97](javascript:void(0);), [110](javascript:void(0);)), enzyme error rates can rise as enzymes become sloppier and encounter higher concentrations of noncanonical substrates in stressed cells. The homeothermic and homeohydric lifestyle of mammals largely insulates them from this type of problem.

A few connections between metabolite damage and the genome or epigenome have been evident for some time, but other, less obvious ones are now emerging. A classical connection is oxidative damage to nucleotides, giving rise to, e.g., 8-oxo-dGTP, which causes mispairing mutations when incorporated into DNA. This damage to the genome is preempted by the Nudix enzyme MutT, a nucleoside-triphosphate pyrophosphohydrolase that is specific for 8-oxo-dGTP and thus “sanitizes” the nucleotide pool ([4](javascript:void(0);), [114](javascript:void(0);)). Two new examples concern the respiratory metabolite fumarate and its spontaneous adduct with glutathione. Fumarate itself is an analog of 2-oxoglutarate and can competitively inhibit 2-oxoglutarate-dependent enzymes, including those that demethylate DNA and histones and hence regulate the epigenome ([76](javascript:void(0);)). Formation of the fumarate-glutathione adduct can deplete the glutathione pool, triggering oxidative stress and consequent DNA damage ([39](javascript:void(0);), [96](javascript:void(0);), [117](javascript:void(0);)). Such small-molecule damage knock-on to the DNA level has prompted speculation that metabolism is far more important in evolution (natural and engineered) than traditional thinking suggests ([14](javascript:void(0);)). If DNA is only stable when metabolite damage is contained, then metabolite damage-control systems would rank alongside the classical DNA damage-repair systems in significance to the genome.

---

## SUMMARY POINTS

| 1.   | Many metabolites are constantly subject to damage in vivo by spontaneous chemical reactions and by side reactions of promiscuous enzymes. |
| ---- | ---------------------------------------- |
| 2.   | Chemical and enzymatic metabolite damage products are wasteful or harmful and are dealt with by biochemical systems that repair or preempt damage; these systems substantially affect fitness and are often widely conserved. |
| 3.   | Metabolite damage reactions and damage-control systems have long been overlooked but are now being uncovered at an increasing rate, not least in plants. |
| 4.   | Many of the thousands of unidentified peaks in plant metabolomic profiles are likely to be damaged metabolites. |
| 5.   | It is likely that hundreds of genes of unknown function in plant genomes encode enzymes involved in metabolite damage-control systems. |
| 6.   | Understanding metabolite damage depends on the systematic integration of insights from chemistry, metabolomics, cheminformatics, biochemistry, and comparative genomics. |